src/Semantic/DataValues/CoordinateValue.php

```diff
@@ -127,7 +76,8 @@ protected function parseUserValue( $value ) {
 
 		if ( $value === '' ) {
 			$this->addError( wfMessage( 'smw_novalues' )->text() );
-		} else {
+		}
+		else {
```

Comment:
Any particular reason to diverge from the common MediaWiki style of not adding newline before else/elseif?

Reply:
This form seemed more common in the code written by the WMDE FUN team. I used the style created for those projects (https://github.com/wmde/intellij-settings) here. Then again I might have picked the wrong option. Still hammering out the IDE rules.

MWs PHPCS does not complain about either form, and perhaps it does not matter that much.